### ISSUE TITLE
Does size of the ad affect revenue?

### ISSUE THREAD

**Issue by belkinvin** [duyduong/flutter_native_admob#66]
As far as I have seen Native ads height can be 90-400. Does having NativeAdmobType.full and NativeAdmobType.banner have difference in revenue?

**Comment by cmd-dev** [comment]
Same question.

**Comment by houdayec** [comment]
A difference would make sense. Usually banners have a lower eCPM than interstitial / bigger ads with image.

Banners < Big Sized Ads < Interstitial < Rewarded Videos

**Comment by belkinvin** [comment]
Yeah but the Native ad specified in this plugin has two types NativeAdmobType.full and NativeAdmobType.banner. This banner ad isn't the same as "Banner Ads" as it uses the adunitID of a Native ad. 
So what exactly is "NativeAdmobType.banner" is it a "Native ad" or a "Banner ad" .
 If ".full and .banner" are both native ads, then will there be any difference in revenue?

**Comment by houdayec** [comment]
I would say banner will generate less revenue as you display less published details. The click rate is lower on banners, less visible so less paid than native ads where you can choose how to optimize visibility and click rate.
